rocketpy/Rocket.py

```diff
@@ -523,6 +523,12 @@ def addFins(self, n, span, rootChord, tipChord, distanceToCM, radius=0):
         radius = self.radius if radius == 0 else radius
         d = 2 * radius
 
+        # Save geometric parameters for later Fin Flutter Analysis 
+        self.rootChord = Cr
```

Comment:
This works if only a single set of fins is added. Document this behavior and propose a method that works with multiple sets of fins.

Reply:
I will do it!